docs/user/vsphere/privileges.md

```diff
@@ -0,0 +1,52 @@
+# Required Privileges
+In order to install an OpenShift cluster to a vCenter, the user provided to the installer needs privileges to read and create the necessary resources. The easiest way to achieve this level of permission and ensure success is to install with a user who has administrative privileges.
+
+Keep in mind that privileges are only required for install (or destroy, if necessary) and can be [revoked after an installation](post.md); they are not necessary for the continued operation of a cluster.
+
+If the provided user has admin privileges, no action is required and you can skip to the next step. Otherwise, the rest of this document can be used as a resource to create a user with more fine-grained privileges.
+
+## Privileges
+In order to create an OpenShift cluster, a user needs permissions for the following categories: _Datastore_, _Folder_, _Host_, _vSphere Tagging_, _Network_, _Resource_, _Profile-driven storage_, _vApp_, and _Virtual machine_.
+
+Here is an example summary of privileges that could be used to install a cluster:
+
+- __Datastore__
+  - _Allocate space_
+- __Folder__
+  - _Create folder_
+  - _Delete folder_
+- __Host__
+  - All privileges
```

Comment:
During 4.6 we should get more clarification (I can help) with what permissions are required.  Host - all privileges seems very broad.

Reply:
I was able to remove these permissions entirely and successfully install. This must have slipped through my trial-and-error process. I am running another test now, just to make sure.